Source/WebKit/WebProcess/WebProcess.cpp

```diff
@@ -1270,7 +1270,7 @@ void WebProcess::networkProcessConnectionClosed(NetworkProcessConnection* connec
     ASSERT_UNUSED(connection, m_networkProcessConnection == connection);
 
     for (auto key : copyToVector(m_storageAreaMaps.keys())) {
-        if (auto map = m_storageAreaMaps.get(key))
+        if (RefPtr map = m_storageAreaMaps.get(key).get())
```

Comment:
do we really need the `.get()` here?

Reply:
I believe we do. Is there some other safe way to convert a WeakPtr to a RefPtr?

Reply:
Oh we could do `releaseImpl()` too. That's probably better right?

Reply:
Did you try?

Reply:
I'm not sure why we're talking about a WeakPtr here.

The map is:
```
HashMap<WebCore::SecurityOriginData, Ref<StorageAreaMap>> m_storageAreaMaps;
```

The value is a `Ref`, when you call `get()` on the HashMap, you'd get a `RefPtr`. So the type already matches, and I suspect we don't need `.get()`

Reply:
The map is on `WebProcess`:

```
HashMap<StorageAreaMapIdentifier, WeakPtr<StorageAreaMap>> m_storageAreaMaps;
```

Reply:
No, I'm certainly not switching from WeakPtr to RefPtr. I must have looked up the wrong HashMap somehow due to similar naming. The one I was looking at had a RefPtr as value. Ignore my comment then.